tests/specs/constructor.js

```diff
@@ -13,3 +14,12 @@ test('returns hoodie.store-inspired API', function (t) {
   t.is(typeof store, 'object', 'is object')
   t.is(store.db, db, 'exposes db')
 })
+
+test('creates store with custom EventEmitter instance', function (t) {
+  t.plan(1)
+
+  var db = dbFactory()
+  var store = db.hoodieSync({emitter: new EventEmitter()})
+
+  t.is(typeof store, 'object', 'is object')
+})
```

Comment:
I'd test that the emitter is in fact used. Could we do something like this? Makes sense?

```js
t.plan(1)

var db = dbFactory()
var emitter = new EventEmitter()
var store = db.hoodieSync({emitter: emitter})

store.on('foo', function() {
  t.ok('emmitter used from options')
})
emitter.emit('foo')
```